crates/ide-completion/src/completions/pattern.rs

```diff
@@ -151,7 +151,6 @@ pub(crate) fn complete_pattern_path(
                     };
 
                     if let Some(hir::Adt::Enum(e)) = ty.as_adt() {
-                        cov_mark::hit!(enum_plain_qualified_use_tree);
```

Comment:
This seems redundant as the check https://github.com/rust-lang/rust-analyzer/blob/09ac44c078dcbad6c19be0430cd9e1248010778d/crates/ide-completion/src/tests/use_tree.rs#L163 is hitted by https://github.com/rust-lang/rust-analyzer/blob/439a513efb77b956ed318cef6a4af421054d29e8/crates/ide-completion/src/completions/use_.rs#L81